### DISCUSSION TITLE
Support For Teams

### DISCUSSION THREAD

**Discussion by botcbmx** [directus/directus#14180] [Archive]
Add suporte for teams, like Laravel Jetstream - https://jetstream.laravel.com/2.x/features/teams.html

Exemplo in Directus: one user can be member of any teams with distinct roles.

Member X:
    Team 1:
     - Role 1
     Team 2:
     - Role 2

And add this in variable filters.

Related Discussion https://github.com/directus/directus/discussions/9682

Thanks..!

**Comment by rijkvanzanten** [comment]
Please upvote https://github.com/directus/directus/discussions/4959 🙂
